zeppelin-server/src/main/java/org/apache/zeppelin/rest/InterpreterRestApi.java

```diff
@@ -90,6 +90,9 @@ public Response newSettings(String message) {
     try {
       NewInterpreterSettingRequest request =
           gson.fromJson(message, NewInterpreterSettingRequest.class);
+      if (request == null) {
+        return new JsonResponse<>(Status.BAD_REQUEST).build();
```

Comment:
@kavinkumarks Yes, I also agree your code to return BAD_REQUEST and to change docs.